airflow/cli/cli_config.py

```diff
@@ -865,7 +865,7 @@ def string_lower_type(val):
 # kubernetes cleanup-pods
 ARG_NAMESPACE = Arg(
     ("--namespace",),
-    default=conf.get("kubernetes_executor", "namespace"),
+    default=conf.get("kubernetes_executor", "namespace", fallback="default"),
```

Comment:
I think you can't remove those defaults. This is a bit chicken-egg problem. At this moment (when parsing args) we do not have providers configuration loaded YET. Loading provider confguration takes few 100 milliseconds and that visibly eats into any cli command if we are loading providers when we just want to figure out what are the command line parameters. So here, fallback **should** remain hard-coded even if this is technically duplication.

Reply:
Turns out we can just remove these commands all together - they are moved to the provider and deprecated in core.